File: cs/auth/1.public/auth/model/user_info.cs
using Microsoft.EntityFrameworkCore.Metadata;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;


namespace HyperId.SDK.Authorization
{
    public struct WalletInfo
    {
        public WalletInfo(string walletAddress,
            string? walletChainId,
            int walletSourceId,
            bool isWalletVerified,
            int walletFamilyId,
            string? walletTags)
        {
            WalletAddress = walletAddress;
            WalletChainId = walletChainId;
            WalletSourceId = walletSourceId;
            IsWalletVerified = isWalletVerified;
            WalletFamilyId = walletFamilyId;
            WalletTags = walletTags;
        }

        public string WalletAddress { get; }
        public string? WalletChainId { get; }
        public int WalletSourceId { get; }
        public bool IsWalletVerified { get; }
        public int WalletFamilyId { get; }
        public string? WalletTags { get; }
    }

    /// <summary>
    /// 
    /// </summary>
    public struct UserInfo
    {
        public UserInfo(string userId,
            bool isGuest,
            string? email,
            bool isEmailVerified,
            string? deviceId,
            string? ip,
            WalletInfo? walletInfo)
        {
            UserId = userId;
            IsGuest = isGuest;
            Email = email;
            IsEmailVerified = isEmailVerified;
            DeviceId = deviceId;
            Ip = ip;
            WalletInfo = walletInfo;
        }

        public string UserId { get; }
        public bool IsGuest { get; }
        public string? Email { get; }
        public bool IsEmailVerified { get; }
        public string? DeviceId { get; }
        public string? Ip { get; }
        public WalletInfo? WalletInfo { get; }
    }
}//HyperId.SDK.Authorization

File: android/hyperid_sdk/src/main/java/ai/hyper_id/sdk/api/auth/model/UserInfo.kt
package ai.hyper_id.sdk.api.auth.model

//**************************************************************************************************
//	WalletInfo
//--------------------------------------------------------------------------------------------------
data class WalletInfo(val walletAddress		: String?		= null,
					  val walletChainId		: String?		= null,
					  val walletSourceId	: Int?			= null,
					  val isWalletVerified	: Boolean		= false,
					  val walletFamilyId	: Int?			= null,
					  val walletTags		: String?		= null)

//**************************************************************************************************
//	UserInfo
//--------------------------------------------------------------------------------------------------
data class UserInfo(val userId						: String?			= null,
					val isGuest						: Boolean			= false,
					val email						: String?			= null,
					val isEmailVerified				: Boolean			= false,
					val deviceId					: String?			= null,
					val ip							: String?			= null,
					val walletInfo					: WalletInfo?		= null)
